fix : Shadow between Toolbar and TabLayout

Shailesh351 opened the pull request:
Fixes #1886

Changes:
- Fix Shadow between Toolbar and TabLayout
- Add elevation in Settings Activity actionbar
- Use dp instead of Px while setting elevation

iamareebjamal commented on android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java:
```diff
@@ -495,6 +495,12 @@ private void replaceFragment(Fragment fragment, int title) {
         }
 
+        if (currentMenuItemId != R.id.nav_schedule) {
+            addShadowToAppBar(true);
```
`addShadowToAppBar(currentMenuItemId != R.id.nav_schedule)`

iamareebjamal commented on android/app/src/main/java/org/fossasia/openevent/activities/SettingsActivity.java:
```diff
@@ -162,6 +163,9 @@ private void setToolbar() {
                 ActionBar.DISPLAY_HOME_AS_UP | ActionBar.DISPLAY_SHOW_TITLE);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
+            getSupportActionBar().setElevation(12);
```
This should be converted to dp to appear normal on each device

Shailesh351 committed:
fix : Shadow between Toolbar and TabLayout

fix : Add elevation in Settings Activity actionbar
fix : Set elevation using Dp
